### PR TITLE
Fix funky css

### PR DESCRIPTION
**Pull request by btzr-io**
## Todo
- [x] Load themes
- [x] Fix `button` style
- [x] Fix `dropdown/input` style
- [x] Fix `tabs` style
- [x] Fix `Navbar` style
- [x] Fix indent issue ( my bad )

### Fix https://github.com/butterproject/butter-desktop/issues/656
I've read the [Contributor License Agreement](/CONTRIBUTING.md#contributor-license-agreement)

### CODE REVIEW

**Comment by btzr-io** [comment]
https://github.com/buttercomponents/butter-base-components/commit/5aca8babf3e1ef461124066087d4a5fab0cdfd00

**Comment by btzr-io** [comment]
https://github.com/buttercomponents/butter-base-components/pull/3

File: src/app/index.html [hunk 1]
```diff
@@ -9,8 +9,9 @@
 	<link rel="stylesheet" type="text/css" href="css/animation.css">
 	<link rel="stylesheet" type="text/css" href="../../node_modules/butter-component-settings/dist/styles.css">
 	<link rel="stylesheet" type="text/css" id="theme" href="themes/Official_-_Dark_theme.css">
+    <link rel="stylesheet" type="text/css" id="NewTheme" href="../../node_modules/butter-theme-dark/index.css">
```
Comment (xaiki) [inline src/app/index.html]: ident is off

Comment (btzr-io) [inline src/app/index.html]: all indent is wrong: https://github.com/butterproject/butter-desktop/blob/xaiki/react/src/app/index.html#L12

**Pull request merged**